### DISCUSSION TITLE
Deleting temporary tables before error on complete-atomic

### DISCUSSION THREAD

**Discussion by rmmhd** [osalvador/ReplicaDB#115] [Ideas]
Hey,

your software works fine. There is one issue with the complete-atomic replication. If an error occurs, the created temporary table still exits. And in my case, I am running replications on multiple tables. So if an error occurs, the database gets filled with many tables. I think it is because the deletion is running after the error occurs.  It would be nice, when the temporary table gets deleted if an error occurs.

**Comment by osalvador** [comment]
Hi @rmmhd 

ReplicaDB delete all temporary tables even if an error occurs. Maybe there is a bug. Could you open an issue with your configuration parameters? use this link please: https://github.com/osalvador/ReplicaDB/issues/new?assignees=&labels=&template=bug_report.md&title=

Thanks.

**Reply by rmmhd** [reply]
Hey @osalvador,

I don't have the configurations anymore. But I am sure the issue occurred when I tried to replicate a view from source by the table name. This didn't work, so I had to change it to a SQL-statement. If this doesn't help, I can try to rebuild the error and open an issue.
